src/guide/list.md

```diff
@@ -1,5 +1,7 @@
 # List Rendering
 
+<VideoLesson href="https://vueschool.io/lessons/list-rendering-in-vue-3?friend=vuejs" title="Learn how to render lists on Vue School">Learn how to render list with a free Vue School lesson</VideoLesson>
```

Comment:
This lesson seemed a little light compared to the CSS class one which goes over the different ways you can bind to a dynamic class. Other scenarios missing include: how looping objects work, pulling out indices, and the recommendation for adding keys to each element to help Vue track of nodes.